cmd/metrics/cmd/mock_dest_store.go

```diff
@@ -50,7 +50,7 @@ func (mds *mockDestStore) Touch(ctx context.Context, objectName string) error {
 var fileListRe *regexp.Regexp
 
 func init() {
-	fileListRe = regexp.MustCompile(`bundle-files-\d+.json$`)
+	fileListRe = regexp.MustCompile(`bundle-files-\d+.yaml$`)
```

Comment:
I think we should be more tolerant here and accept json and support repos created with pre-V2.

Reply:
The old repos will be moved but no need to concurrently support both.

Reply:
ok as per our discussion